docusaurus/docs/advanced-configuration.md

```diff
@@ -3,23 +3,21 @@ id: advanced-configuration
 title: Advanced Configuration
 ---
 
-The `.config/` directory holds the preferred configuration for the different tools
-that are used to develop, test and build the project.
+The `.config/` directory holds the preferred configuration for the different tools used to develop, test and build a plugin.
```

Comment:
insert Oxford comma: "test, and build a plugin"

Reply:
for more info, see https://grafana.com/docs/writers-toolkit/style-guide/capitalization-punctuation/